### PR TITLE
Fix SCIM setup

### PR DESCRIPTION
**Pull request by jwitz**
See https://astronomer.slack.com/archives/CEPE4K68P/p1674573550354179

### CODE REVIEW

**Review by frankcash — APPROVED** [review]

File: software/integrate-auth-system.md [hunk 1]
```diff
@@ -422,17 +422,21 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
+5. In the application management menu for your new application, go to **Manage** > **Provisioning**.
 
-3. Click **Provisioning Mode** > **Automatic**.
+6. Click **Provisioning Mode** > **Automatic**.
 
-4. In the **Tenant URL** field, enter `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
+7. In the **Tenant URL** field, enter `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
    
-5. In your `config.yaml` file, add the following configuration to create a unique token that Astronomer Software uses to authenticate requests from Azure AD:
+8. In your `config.yaml` file, add the following configuration to create a unique token that Astronomer Software uses to authenticate requests from Azure AD:
```
Comment (odaneau-astro) [inline software/integrate-auth-system.md:439]: The customer was asking us where this token is coming from. Is it generated in Azure AD or somewhere else?

**Review by odaneau-astro — CHANGES_REQUESTED** [review]

File: software/integrate-auth-system.md [hunk 2]
```diff
@@ -422,17 +422,21 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
 
 <TabItem value="azuread">
 
-This setup assumes you have already registered Astronomer Software as an application on Azure AD. See [Azure AD](#azure-ad).
-
-1. If user provisioning isn't available on your existing Astronomer Software application, create a new application using the **Azure Databricks SCIM Provisioning Connector**. See [Configure SCIM provisioning using Microsoft Azure Active Directory](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/scim/aad#step-2-configure-the-enterprise-application).
+1. Sign in to the [Azure AD portal](https://aad.portal.azure.com/). 
+   
+2. In the left menu, select **Enterprise applications**, then go to **+ New application** > **+ Create your own application**.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:427]:
```suggestion
2. In the left menu, select **Enterprise applications**, and then click **New application** > **Create your own application**.
```

File: software/integrate-auth-system.md [hunk 3]
```diff
@@ -422,17 +422,21 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
+   
+3. Enter a name for your application and select **Integrate any other application you don't find in the gallery**.
+   
+4. Click **Add** to create an app object. Azure AD opens the application management menu for your new application.
```
Comment (kwcanuck) [inline software/integrate-auth-system.md:431]:
```suggestion
4. Click **Add** to create an application object. 
```
_Azure AD opens the application management menu for your new application._ - if this is obvious behavior and not unexpected, it doesn't need to be described.

**Review by kwcanuck — APPROVED** [review]

File: software/integrate-auth-system.md [hunk 4]
```diff
@@ -422,17 +422,21 @@ See [Add SCIM provisioning to app integrations](https://help.okta.com/en-us/Cont
...
    
-2. In your Astronomer Software application on Azure AD, go to **Manage** > **Provisioning**.
+5. In the application management menu for your new application, go to **Manage** > **Provisioning**.
 
-3. Click **Provisioning Mode** > **Automatic**.
+6. Click **Provisioning Mode** > **Automatic**.
 
-4. In the **Tenant URL** field, enter `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
+7. In the **Tenant URL** field, enter `https://BASEDOMAIN.astronomer.io/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
```
Comment (jwitz) [inline software/integrate-auth-system.md:437]:
```suggestion
7. In the **Tenant URL** field, enter `https://astro.BASEDOMAIN/v1/scim/microsoft`. This is the Astronomer SCIM endpoint URL.
```

File: software/integrate-auth-system.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (odaneau-astro) [reply software/integrate-auth-system.md]: The answer to this question is Yes. You end up with one "Registered Application" to handle OIDC and one "Enterprise Application" to handle SCIM. Maybe we should mention it somewhere.

File: software/integrate-auth-system.md (hunk 1 above)
Comment (odaneau-astro) [reply software/integrate-auth-system.md]: The token is a randomly generated string. As long as it matches between the helm chart and Azure AD, it should just work.

**Comment by odaneau-astro** [comment]
For future reference, our internal documentation on SCIM is in Notion here: https://www.notion.so/astronomerio/SCIM-v2-Integration-3c12fc8255594b20bea199ec8b5e6916

**Review by odaneau-astro — APPROVED** [review]

**Pull request merged**